src/twisted/application/strports.py

```diff
@@ -8,13 +8,23 @@
 @see: L{twisted.internet.endpoints.serverFromString}
 @see: L{twisted.internet.endpoints.clientFromString}
 """
-
+from typing import cast, Optional
 
 from twisted.application.internet import StreamServerEndpointService
-from twisted.internet import endpoints
+from twisted.internet import endpoints, interfaces
+
+
+def _getReactor() -> interfaces.IReactorCore:
+    from twisted.internet import reactor
+
+    return cast(interfaces.IReactorCore, reactor)
```

Comment:
I'm pretty sure we're going to need to add a public `getInstalledReactor()` callable like this, because of this casting requirement, so clients don't have to add this in every project.